src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -9,13 +9,24 @@
 public class SelectTestFigureOptionListener implements ActionListener {
 
 	private DriverManager driverManager;
+	private String figureName;
 
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
+	public SelectTestFigureOptionListener(DriverManager driverManager, String figureName) {
 		this.driverManager = driverManager;
+		this.figureName = figureName;
 	}
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		switch (figureName) {
+			case "figureScript1":
+				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+				break;
+			case "figureScript2":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.